package security.auth.exception;

public class PhoneNumberAlreadyInUseException extends Throwable {
    public PhoneNumberAlreadyInUseException(String s) {
        super(s);
    }
}
